core/files/uploads-manager.php

```diff
@@ -292,7 +292,9 @@ public function get_temp_dir() {
 		if ( ! $this->temp_dir ) {
 			$wp_upload_dir = wp_upload_dir();
 
-			$this->temp_dir = implode( DIRECTORY_SEPARATOR, [ $wp_upload_dir['basedir'], 'elementor', 'tmp' ] ) . DIRECTORY_SEPARATOR;
+			$temp_dir = implode( DIRECTORY_SEPARATOR, [ $wp_upload_dir['basedir'], 'elementor', 'tmp' ] ) . DIRECTORY_SEPARATOR;
+
+			$this->temp_dir = apply_filters( 'elementor/files/temp-dir', $temp_dir );
```

Comment:
```suggestion
			/**
			 * Temp File Path
			 *
			 * Allows modifying the full path of the temporary file.
			 *
			 * @since 3.7.0
			 *
			 * @param string file name
			 */
			$this->temp_dir = apply_filters( 'elementor/files/temp-dir', $temp_dir );
```